fundamentals/fundamentals-bio/src/main/scala/izumi/functional/bio/Clock1.scala

```diff
@@ -39,50 +44,119 @@ object Clock1 extends LowPriorityClockInstances {
+      // do not reuse nowZoned because of sjs
+      ClockAccuracy.applyAccuracy(OffsetDateTime.now(zone), accuracy)
     }
 
-    private[this] final val TZ_UTC: ZoneId = ZoneId.of("UTC")
   }
 
+  @deprecated("Use ConstantZoned/ConstantOffset")
   final class Constant(time: ZonedDateTime, nano: Long) extends Clock1[Identity] {
+    private val underlying = new ConstantZoned(time, nano)
+
+    override def epoch: Long = underlying.epoch
+    @deprecated("use nowZoned")
+    override def now(accuracy: ClockAccuracy): ZonedDateTime = underlying.now(accuracy)
+
+    override def nowLocal(accuracy: ClockAccuracy, zone: ZoneId): LocalDateTime = underlying.nowLocal(accuracy, zone)
+    override def nowOffset(accuracy: ClockAccuracy, zone: ZoneId): OffsetDateTime = underlying.nowOffset(accuracy, zone)
+    override def nowZoned(accuracy: ClockAccuracy, zone: ZoneId): Identity[ZonedDateTime] = underlying.nowZoned(accuracy, zone)
+    override def monotonicNano: Long = nano
+  }
+
+  final class ConstantZoned(time: ZonedDateTime, nano: Long) extends Clock1[Identity] {
+    override def epoch: Long = time.toEpochSecond
+    @deprecated("use nowZoned")
+    override def now(accuracy: ClockAccuracy): ZonedDateTime = nowZoned(accuracy)
+
+    override def nowLocal(accuracy: ClockAccuracy, zone: ZoneId): LocalDateTime = ClockAccuracy.applyAccuracy(time.toLocalDateTime, accuracy)
+    override def nowOffset(accuracy: ClockAccuracy, zone: ZoneId): OffsetDateTime = ClockAccuracy.applyAccuracy(time.toOffsetDateTime, accuracy)
+    override def nowZoned(accuracy: ClockAccuracy, zone: ZoneId): Identity[ZonedDateTime] = ClockAccuracy.applyAccuracy(time, accuracy)
+    override def monotonicNano: Long = nano
+  }
+
+  final class ConstantOffset(time: OffsetDateTime, nano: Long) extends Clock1[Identity] {
     override def epoch: Long = time.toEpochSecond
-    override def now(accuracy: ClockAccuracy): ZonedDateTime = ClockAccuracy.applyAccuracy(time, accuracy)
-    override def nowLocal(accuracy: ClockAccuracy): LocalDateTime = now(accuracy).toLocalDateTime
-    override def nowOffset(accuracy: ClockAccuracy): OffsetDateTime = now(accuracy).toOffsetDateTime
+    @deprecated("use nowZoned")
+    override def now(accuracy: ClockAccuracy): ZonedDateTime = nowZoned(accuracy)
+
+    override def nowLocal(accuracy: ClockAccuracy, zone: ZoneId): LocalDateTime = ClockAccuracy.applyAccuracy(time.toLocalDateTime, accuracy)
+    override def nowOffset(accuracy: ClockAccuracy, zone: ZoneId): OffsetDateTime = ClockAccuracy.applyAccuracy(time, accuracy)
+    override def nowZoned(accuracy: ClockAccuracy, zone: ZoneId): Identity[ZonedDateTime] = ClockAccuracy.applyAccuracy(time.toZonedDateTime, accuracy)
     override def monotonicNano: Long = nano
   }
 
   sealed trait ClockAccuracy
   object ClockAccuracy {
+    @deprecated("Use ClockAccuracy.RAW (but better set the limit explicitly!)")
     case object DEFAULT extends ClockAccuracy
```

Comment:
Remove from public scope
```suggestion
    private[izumi] case object DEFAULT extends ClockAccuracy
```